crates/python/pyproject.toml

```diff
@@ -29,14 +29,21 @@ version = "0.8.0"
 description = "Python interface for the QCS Rust SDK"
 readme = "README.md"
 authors = [
-	"Rigetti Computing <softapps@rigetti.com>",
-	"Mark Skilbeck <mark.skilbeck@rigetti.com>",
-	"Marquess Valdez <mvaldez@rigetti.com>",
-	"Randall Fulton <rfulton@rigetti.com>",
+  "Rigetti Computing <softapps@rigetti.com>",
+  "Mark Skilbeck <mark.skilbeck@rigetti.com>",
+  "Marquess Valdez <mvaldez@rigetti.com>",
+  "Randall Fulton <rfulton@rigetti.com>",
 ]
 
+[tool.maturin]
+features = ["pyo3/extension-module"]
+bindings = "pyo3"
+compatibility = "linux"
+sdist-include = ["README.md"]
+
 [tool.poetry.dependencies]
 python = "^3.8"
+quil = "0.1.0-rc.0"
```

Comment:
Technically the same code as is in [`0.1.1`](https://pypi.org/project/quil/0.1.1/) right?

Reply:
yeah, but should've been 0.1.1